docs/index.md

```diff
@@ -58,6 +58,13 @@ You will need
 
 The database we use is PostgreSQL 10, but you do not need to install that - it's installed via a Docker image. RStudio 1.2 is highly recommended but not required.
 
+In addition to the current version of R and Rstudio, you will need the following packages:
+
+* tidyverse
+* DBI
+* RPostgres
+* glue
+
```

Comment:
The last time I attempted to knit it made me install `skimr`. So I think we need that package here too.

Reply:
I searched for `library(` and found the following statements, so I assume all of those packages need to be specified up front:
* library(tidyverse)
* library(DBI)
* library(RPostgres)
* library(glue)
* library(downloader, here)
* library(skimr)
* library(DT)

Not sure if this needs to be included, since it is in the `potential_architectures.Rmd`:
* library(DiagrammeR)